deployment/helm/balloons/crds/config.nri_balloonspolicies.yaml

```diff
@@ -295,6 +295,54 @@ spec:
                   IdleCpuClass controls how unusded CPUs outside any a
                   balloons are (re)configured.
                 type: string
+              ignore:
+                description: |-
+                  Ignore specifies containers that should be completely
+                  ignored by the policy.
+                properties:
+                  matchExpressions:
+                    description: MatchExpressions specifies one or more expressions.
+                    items:
+                      description: |-
+                        Expression describes some runtime-evaluated condition. An expression
+                        consist of a key, an operator and a set of values. An expressions is
+                        evaluated against an object which implements the Evaluable interface.
+                        Evaluating an expression consists of looking up the value for the key
+                        in the object, then using the operator to check it agains the values
```

Comment:
```suggestion
                        in the object, then using the operator to check it against the values
```

Reply:
Another nit, I suspect this description was added manually here since it doesn't exist in the field description in pkg/apis/config/v1alpha1/resmgr/policy/balloons/config.go file? Should we perhaps add this description there and let the code generator to take care of adding the same description to the CRD here? Feel free to not do it since this is not a showstopper.

Reply:
Definitely. The resource management plugin CRDs should never be manually edited. Instead the corresponding golang types should be updated (and annotated with the necessary comments) then CRD (among the whole shebang) regenerated by `make gneerate`).

Reply:
No, I think @askervin did it correctly here, in this PR...

That description properly propagates from `pkg/apis/resmgr/v1alpha1/types.go` where `Expression`, which is only referenced here, is originally defined.

Reply:
Also, if I run `make generate && git status --short` everything looks clean.

Reply:
Ah true, [Expression](https://github.com/containers/nri-plugins/blob/5d4ed7c4defcbc30a1c10eb0d91887ca055a1419/pkg/apis/resmgr/v1alpha1/types.go#L17) definition is already there on the main. I thought it is being added here instead of referencing to the something that was already there. Then all good here.